docker-compose.yml

```diff
@@ -8,16 +8,56 @@ services:
       - "5432:5432"
     environment:
       POSTGRES_USER: ${USER}
+    networks:
+      - entropic
+
   redis:
     image: redis:alpine
     ports:
       - "6379:6379"
+    networks:
+      - entropic
+
   queue:
     image: schickling/beanstalkd
     ports:
       - "11300:11300"
     volumes:
       - beanstalk_wal:/var/lib/beanstalkd/
+    networks:
+      - entropic
+
+  registry:
+    image: mhart/alpine-node:12
+    environment:
+      NODE_ENV: development
+      POSTGRES_URL: postgres://postgres@db:5432
+      POSTGRES_HOST: db
+      REDIS_URL: redis://redis:6379
+      PORT: 3000
+      DEV_LATENCY_ERROR_MS: 10000
+      PGUSER: postgres
+      PGDATABASE: entropic_dev
+      CACHE_DIR: ../data/entropic-cache
+      OAUTH_GITHUB_CLIENT: gh_client_id_here
+      OAUTH_GITHUB_SECRET: gh_secret_here
+      OAUTH_PASSWORD: pw_for_encrypting_tokens_here
```

Comment:
A docker-compose newbie q: would it be possible to use `env_file` here instead? (A la [this doc](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option))

Reply:
Yeah absolutely, I'll change that!